cobalt/worker/worker_consts.h

```diff
@@ -18,32 +18,61 @@
 namespace cobalt {
 namespace worker {
 
-struct WorkerConsts {
-  // Constants for error messages.
-  static const char kServiceWorkerRegisterBadMIMEError[];
-  static const char kServiceWorkerRegisterNoMIMEError[];
-  static const char kServiceWorkerRegisterScriptOriginNotSameError[];
-  static const char kServiceWorkerRegisterScopeOriginNotSameError[];
-  static const char kServiceWorkerRegisterBadScopeError[];
-  static const char kServiceWorkerUnregisterScopeOriginNotSameError[];
-  static const char kServiceWorkerAllowed[];
-
-  // Constants for ServiceWorkerPersistentSettings
-  static const char kSettingsJson[];
-
-  // Array of JavaScript mime types, according to the MIME Sniffinc spec:
-  //   https://mimesniff.spec.whatwg.org/#javascript-mime-type
-  static const char* const kJavaScriptMimeTypes[16];
-
-  // The name of a Service Worker thread.
-  static const char kServiceWorkerName[];
-
-  // The name of a Service Worker Registry thread.
-  static const char kServiceWorkerRegistryName[];
-
-  // The name of a Dedicated Worker thread.
-  static const char kDedicatedWorkerName[];
-};
+namespace WorkerConsts {
```

Comment:
Renaming struct to namespace lets calling code still to use `worker::WorkerConsts` to refer to the names. This leaves client code untouched